static/scss/answers/module.scss

```diff
@@ -59,10 +44,8 @@
     justify-content: flex-end;
     align-items: flex-end;
     flex-grow: 1;
-    @include bpgte(sm)
-    {
-      margin-top: calc(var(--yxt-base-spacing) / 2);
-    }
+    margin-top: calc(var(--yxt-base-spacing) / 2);
+    height: auto;
```

Comment:
just checking that we need to specify height: auto?

Reply:
you're right, I needed it in a previous version of this PR, but it's not needed anymore